WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -412,16 +216,21 @@ private extension TotalsViewModel {
 
     var presentationStyleDeterminerDependencies: PointOfSaleCardPresentPaymentEventPresentationStyle.Dependencies {
         let cancelThenCollectPaymentWithWeakSelf: () -> Void = { [weak self] in
-            self?.cancelThenCollectPayment()
+            self?.posModel.cancelThenCollectPayment()
+        }
+
+        var orderTotal: String?
+        if case .loaded(let totals) = posModel.orderState {
+            orderTotal = totals.orderTotal
         }
 
         return PointOfSaleCardPresentPaymentEventPresentationStyle.Dependencies(
             tryPaymentAgainBackToCheckoutAction: cancelThenCollectPaymentWithWeakSelf,
             nonRetryableErrorExitAction: cancelThenCollectPaymentWithWeakSelf,
-            formattedOrderTotalPrice: formattedOrderTotalPrice,
+            formattedOrderTotalPrice: orderTotal,
             paymentCaptureErrorTryAgainAction: cancelThenCollectPaymentWithWeakSelf,
             paymentCaptureErrorNewOrderAction: { [weak self] in
-                self?.startNewOrder()
+                self?.posModel.startNewCart()
```

Comment:
Before this PR, this set of dependencies stopped me moving the payment states to the aggregate model. It needed the order total and couldn't get it until it was responsible for syncing the order.

In the next PR (I think), I'll move this to the aggregate model along with all the payment modals/event processing.